terms.html

```diff
@@ -1,203 +1,209 @@
 The portion of a <a>DID URL</a> that begins with and includes the first forward
-slash character (<code>/</code>). DID path syntax is identical to the URI path
-syntax.
+slash (<code>/</code>) character and ends with either a question mark (<code>?</code>)
+character or a fragment hash sign (<code>#</code>) character (or the end of the
+DID URL). DID path syntax is identical to URI path syntax. See
+<a href="#did-path"></a>.
   </dd>
 
   <dt><dfn data-lt="DID queries">DID query</dfn></dt>
 
   <dd>
-The portion of a <a>DID URL</a> that follows the first question mark character
-(<code>?</code>). DID query syntax is identical to the URI query syntax.
-  </dd>
-
-  <dt><dfn data-lt="verifiable data registry|verifiable data registries">Verifiable
-  data registry</dfn></dt>
-
-  <dd>
-A role a system performs to mediate the creation, verification, updating, and
-deactivation of <a>decentralized identifiers</a>. For more information, see
-[[VC-DATA-MODEL]].
+The portion of a <a>DID URL</a> that follows and includes the first question
+mark character (<code>?</code>). DID query syntax is identical to URI query
+syntax. See <a href="#did-query"></a>.
   </dd>
 
   <dt><dfn>DID resolution</dfn></dt>
 
   <dd>
 The function that takes as its input a <a>DID</a> and a set of input
-metadata and returns a <a>DID document</a> in a conforming representation
+metadata and returns a <a>DID document</a> in a conforming <a>representation</a>
 plus additional metadata. This function relies on the "Read" operation of the
-applicable <a>DID method</a>. The inputs and outputs of this function are defined
-in <a href="#did-resolution"></a>.
+applicable <a>DID method</a>. The inputs and outputs of this function are
+defined in <a href="#did-resolution"></a>.
   </dd>
 
   <dt><dfn data-lt="DID resolvers">DID resolver</dfn></dt>
 
   <dd>
-A <a>DID resolver</a> is a software and/or hardware component that takes
-a <a>DID</a> as input and produces a conforming <a>DID document</a> as
-output by performing <a>DID resolution</a>.
+A <a>DID resolver</a> is a software and/or hardware component that performs the
+<a>DID resolution</a> function by taking a <a>DID</a> as input and producing a
+conforming <a>DID document</a> as output.
   </dd>
 
   <dt><dfn data-lt="DID schemes">DID scheme</dfn></dt>
 
   <dd>
 The formal syntax of a <a>decentralized identifier</a>. The generic DID scheme
-is defined in the <a href="https://w3.org/TR/did-core">DID Core specification</a>.
-Separate <a>DID method</a> specifications define a specific DID scheme that
-works with that specific <a>DID method</a>.
+begins with the prefix <code>did:</code> as defined in the <a href="did-syntax"></a>
+section of the <a href="https://w3.org/TR/did-core">DID Core specification</a>.
+Each <a>DID method</a> specification must define a specific DID
+scheme that works with that specific <a>DID method</a>. In a specific DID method
+scheme, the DID method name must follow the first colon and terminate with the
```

Comment:
@kdenhartog You're right that we do not currently have a formal definition of a "submethod name". So it's an open issue whether we want to formalize that or not. What's your feeling about that? Do you think we should make it a formal concept in the spec? And reflect that in how we structure the ABNF? (Which BTW would not be any normative change to the syntax, just how how we talk about it.)

Reply:
I'm thinking it will be useful to define them. Having that language in place in my mind will be helpful when describing the network of networks approach as well as ways that the identifiers can be modified to address forking concerns. Even if we don't explicitly mention them within the text now, I think by defining the term we'll seed the idea of being able to use that technique into DID Method author's mind in the which will allow it to be used in creative ways.

This could likely be done as a separate issue though and doesn't need to block this in order for it to be completed.